protobuf/model_config.proto

```diff
@@ -130,6 +130,44 @@ message ModelRateLimiter
   uint32 priority = 2;
 }
 
+//@@
+//@@  .. cpp:var:: message SecondaryDevice
+//@@
+//@@     The secondary device to run the model instances on. Not all backend require this. For the
```

Comment:
Lets keep it general as discussed. Don't mention DLAcore or TRT backend here. We can include this warning just above KIND_DLA.